gl_extensions/include/core/gl_extensions.hpp

```diff
@@ -19,20 +12,13 @@
 // GL_ARB_shader_objects, GL_ARB_vertex_program, GL_ARB_fragment_program, GL_ARB_vertex_shader, GL_ARB_fragment_shader
 // GL_ARB_sync
 // GL_ARB_vertex_array_object
-// WGL_ARB_extensions_string
-// WGL_ARB_pixel_format
-// WGL_ARB_create_context
-// WGL_EXT_swap_control
-//
-//  AUTHOR: Song Ho Ahn (song.ahn@gmail.com)
-// CREATED: 2013-03-05
-// UPDATED: 2018-08-09
 ///////////////////////////////////////////////////////////////////////////////
 
 #ifndef GL_EXTENSION_H
 #define GL_EXTENSION_H
 
 // in order to get function prototypes from glext.h, define GL_GLEXT_PROTOTYPES before including glext.h
+//TODO: find the way to replace this with a function call
```

Comment:
Remove this TODO, since in the end all of the functions are initialized by the means of calling a function.

Reply:
Done